### ISSUE TITLE
Create a button to increase contrast for WCAG

### ISSUE THREAD

**Issue by jolevesq** [fgpv-vpgf/fgpv-vpgf#1810]
Some icon are hard to see for people with eye disabilities. We need a button like Touch Mode who increase button size to increase contrast so text and icons are easy to see

**Comment by mweech** [comment]
is this because GcViz has it? or is there some test to validate its need?

**Comment by AleksueiR** [comment]
Which buttons don't have sufficient contrast?
We should make sure the contrast is proper in the default view instead of providing a separate mode.
I think browser zoom should be adequate if you want to increase the size of buttons and other elements.

**Comment by jolevesq** [comment]
All the map nav button with the transparency. When you focus it becomes white but it can be hard to see before we focus on it. Maybe some of them in out toc like ... and the eye.

Someone from the user experience team told me that. I will ask him to give the problem with actual contrast ratio and what it should be.

**Comment by AleksueiR** [comment]
If it's only the nav buttons, we can remove opacity from them. The `eye` icon in layers will change to checkbox/radio button in 2.0 in support of the structured legend.

If they can provide us specific elements that don't meet contrast requirement, that would help greatly.

**Comment by jolevesq** [comment]
Follow, there is a list of all the icons and their contrast value. They need to pass aaa.

<img width="490" alt="screen shot 2017-03-28 at 3 01 43 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422558/f04397a2-13c7-11e7-8336-01c68040aa4b.png">
<img width="450" alt="screen shot 2017-03-28 at 3 02 08 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422559/f2cbeaec-13c7-11e7-9174-c093b892c87d.png">
<img width="461" alt="screen shot 2017-03-28 at 3 02 19 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422569/f6f07de0-13c7-11e7-8e4d-399f744c0832.png">
<img width="437" alt="screen shot 2017-03-28 at 3 02 30 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422572/f93b2e38-13c7-11e7-96f5-3ffcb12d5177.png">
<img width="439" alt="screen shot 2017-03-28 at 3 02 42 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422573/fb40fd3e-13c7-11e7-9ccf-84cd5c988264.png">
<img width="427" alt="screen shot 2017-03-28 at 3 02 54 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422575/fd15d1de-13c7-11e7-81c5-d633b5bf5ae6.png">
<img width="449" alt="screen shot 2017-03-28 at 3 03 08 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422576/ff3588ba-13c7-11e7-9111-a1b4d608f539.png">
<img width="459" alt="screen shot 2017-03-28 at 3 03 22 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422578/01246114-13c8-11e7-8f02-507e028aac3c.png">
<img width="434" alt="screen shot 2017-03-28 at 3 03 32 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422583/045b1814-13c8-11e7-8069-71243c9f78f4.png">
<img width="776" alt="screen shot 2017-03-28 at 3 03 41 pm" src="https://cloud.githubusercontent.com/assets/3472990/24422584/06a70c7c-13c8-11e7-8b3b-fb915ef7d72a.png">

**Comment by AleksueiR** [comment]
- We are targeting WCAG AA compliance, not AAA.
- This contrast checker is for text, not icons (they are larger than regular text and have different meanings)
